e2e/pod.go

```diff
@@ -30,6 +33,10 @@ func ensurePod(ctx context.Context, s *Scenario, pod *corev1.Pod) {
 
 	_, err = kube.WaitUntilPodRunning(ctx, s.T, pod.Namespace, "", "metadata.name="+pod.Name)
 	require.NoErrorf(s.T, err, "failed to wait for pod %q to be in running state", pod.Name)
+
+	timeForReady := time.Since(start)
+	fmt.Printf("##vso[task.logissue type=warning;]Time for pod %q to get ready was %s\n", pod.Name, timeForReady)
```

Comment:
same, where is this a warning ?

Reply:
Have changed to only log as a warning if the duration is over a limit.